### ISSUE TITLE
isJSON fails to detect some JSON http responses

### ISSUE THREAD

**Issue by ghost** [domoticz/domoticz#3929]
Hi all,

Querying DarkSky it answers with a json response whose header content-type field is: "application/json; charset=utf-8" . Seems that the appended "; charset=utf-8" fools isJSON to not detect it as a valid JSON response. Thus, fields cannot be extracted from item.json.

Note that dz.utils.fromJSON(item.data, nil) is perfectly able to return a list whose fields can be extracted.

Domoticz: 4.10717
dzVents: 2.4.19
Python: 3.7.3

![DarkSky_JSON](https://user-images.githubusercontent.com/6748821/74937960-6e3bf880-53ed-11ea-94c3-0db094face84.png)

**Comment by rwaaren** [comment]
The issue is not the the appended "; charset=utf-8" but 
Darksky does not fully comply to RFC-2616
The standard field-name is  "Content-Type"  and case matters.

Now Darksky is not the only party not complying fully to this standard and that's why it was decided in later versions of dzVents to accept Content-Type as valid filed-name regardless the case by converting it to lowercase before trying to match it.

In short. It's solved in later Beta's
